staging/src/k8s.io/apiserver/pkg/util/feature/feature_gate.go

```diff
@@ -127,13 +139,23 @@ func (f *featureGate) Set(value string) error {
 	f.lock.Lock()
 	defer f.lock.Unlock()
 
+	// Copy existing state
+	known := map[Feature]FeatureSpec{}
+	for k, v := range f.known.Load().(map[Feature]FeatureSpec) {
```

Comment:
It probably isn't necessary to copy `f.known` here, as neither `Set` nor any of the existing `specialFeatures` handlers mutate it. That said, if we stop copying it, a comment here and another near `specialFeatures` is probably a good idea.

Reply:
This is true, but OTOh this should be called pretty rarely and in that case i prefer easier and more readable code that this micro-optimization.